execution_engine_testing/test_support/src/chainspec_config.rs

```diff
@@ -66,6 +72,19 @@ pub struct CoreConfig {
     pub(crate) minimum_delegation_amount: u64,
     /// Enables strict arguments checking when calling a contract.
     pub(crate) strict_argument_checking: bool,
+    /// Auction entrypoints such as "add_bid" or "delegate" are disabled if this flag is set to
+    /// `true`. Setting up this option makes sense only for private chains where validator set
+    /// rotation is unnecessary.
+    pub(crate) allow_auction_bids: bool,
```

Comment:
Isn't this named backwards? I would expect this setting to be
```suggestion
    /// Auction entrypoints such as "add_bid" or "delegate" are disabled if this flag is set to
    /// `true`. Setting up this option makes sense only for private chains where validator set
    /// rotation is unnecessary.
    pub(crate) forbid_auction_bids: bool,
```

Reply:
I think we discussed earlier in the meeting this it is backwards to name it allow but always negate it, so I agree; I went with `disable` as an opposite to `allow, rather than forbid, however

Reply:
After discussing it with @Fraser999, we decided to revert the change to `allow_foo` in 0ec0e34bc7fa9531e836643553c08ae55b01f276 and do a follow-up work post-Wedneday on this.